src/components/nanopub/Nanopub.tsx

```
@@ -0,0 +1,921 @@
+  object,
+  predicate,
+}: {
+  subject: string;
+  object: string;
+  predicate: string;
+}) => (
+  <div className="grid grid-cols-12 gap-4 border-0 border-b sm:border-b-0 py-2">
+    <Text
+      className="col-span-12 sm:col-span-2 truncate"
+      title={subject}
+      children={subject}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)